webapp/app.py

```diff
@@ -198,7 +198,11 @@ def bad_request_error(error):
 
 @app.errorhandler(SecurityAPIError)
 def security_api_error(error):
-    return flask.render_template("security-error.html"), 500
+    message = error.response.json()
```

Comment:
Rather than doing `message.message` in the template, which is kinda clunky, why not extract the  message here instead:

```suggestion
    message = error.response.json().get("message")
```